lib/metadata/ocsp/client.rb

```diff
@@ -69,7 +82,7 @@ def check_response(ocsp_request, ocsp_response, cert_ids, store)
         basic_response = ocsp_response.basic
         fail CheckerError, "could not verify response against issuer certificates" unless basic_response.verify([], store)
         statuses = basic_response.status
-        results = statuses.inject({}) do |hash,status|
+          results = statuses.inject({}) do |hash,status|
```

Comment:
```suggestion
        results = statuses.inject({}) do |hash,status|
```
This looks like a spurious change